sklearn/utils/estimator_checks.py

```diff
@@ -1574,9 +1574,9 @@ def check_parameters_default_constructible(name, Estimator):
         try:
             def param_filter(p):
                 """Identify hyper parameters of an estimator"""
-                return (p.name != 'self'
-                        and p.kind != p.VAR_KEYWORD
-                        and p.kind != p.VAR_POSITIONAL)
+                return (p.name != 'self' and
```

Comment:
For next time, try not to change things that are not related to your PR. This adds noise into the diff and makes it harder for the review to be efficient.